models/app-models.js

```diff
@@ -37,15 +38,23 @@ exports.changeVotes = async (article_id, addVotes = 0) => {
 };
 
 exports.selectUsers = async () => {
-    const getUsers = await db.query(`SELECT username FROM users`)
-    return getUsers.rows
-}
+  const getUsers = await db.query(`SELECT username FROM users;`);
+  return getUsers.rows;
+};
 
 exports.selectAllArticles = async () => {
-    const getArticles = await db.query(`SELECT author, title, article_id, topic, created_at, votes FROM articles ORDER BY created_at DESC`)
-    return getArticles.rows
-}
+  const getArticles = await db.query(
+    `SELECT author, title, article_id, topic, created_at, votes FROM articles ORDER BY created_at DESC;`
+  );
+  return getArticles.rows;
+};
 
-// exports.selectArticleIdComments = async () => {
-//     const 
-// }
+exports.selectArticleIdComments = async (article_id) => {
```

Comment:
doug being nit-picky & subjective:

`selectArticleIdComments` isn't the most descriptive function name here

something like `selectCommentsByArticleId` would have been something i'd go for